ideas/for-an-inclusive-quayside/index.html

```diff
@@ -0,0 +1,111 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <title>Co-designing Inclusive Cities - Ideas for an Inclusive Quayside</title>
+    <meta charset="utf-8">
+
+    <mCo-Designing eta name="viewport" content="width=device-width, initial-scale=1">
```

Comment:
Looks like there's a typo here, making your markup invalid. I think this should be a meta tag?